Index: config.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>token = \"\"\nurl = \"https://gestion.eirlab.net/api/index.php/\"\nheaders = {\n    \"Accept\": \"application/json\",\n    \"DOLAPIKEY\": token}\nurl_member = \"members?sortfield=t.rowid&sortorder=ASC&limit=600\"\nurl_user = \"users?sortfield=t.rowid&sortorder=ASC&limit=100\"\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.py b/config.py
--- a/config.py	(revision e83b9190c4f62a94e1901fc361a020a7352e915e)
+++ b/config.py	(date 1658426941033)
@@ -1,4 +1,4 @@
-token = ""
+token = "08b977403545a0c5eafd4ee04d6ce84105f6c229"
 url = "https://gestion.eirlab.net/api/index.php/"
 headers = {
     "Accept": "application/json",
